RoosterPlanner.Api/Controllers/TasksController.cs

```diff
@@ -337,5 +339,124 @@ public async Task<ActionResult> RemoveCategory(Guid id)
                 return UnprocessableEntity();
```

Comment:
Ik mis hier mogelijk een foutmelding of error object wat aangeeeft wat er fout is gegaan.